charts/dafka-consumer/README.md

```diff
@@ -65,6 +65,8 @@ A Helm Chart for Dafka Consumer
 | podMonitor.sampleLimit | string | `nil` | set sample limit for the pod monitor |
 | pdb.enabled | bool | `false` | Set to true to enable |
 | jvmOpts | string | `nil` |  |
+| prometheusRule.enabled | bool | `false` |  |
```

Comment:
prometheusRules (plural)

Reply:
No because it one object that can have many groups